src/app/authorization/login/login.component.ts

```diff
@@ -12,62 +15,46 @@ import dialogData from 'src/assets/json/dialogData.json';
 export class LoginComponent implements OnInit, OnDestroy {
   userData: any;
   dialogData = { ...dialogData };
-  adminForm!:FormGroup;
+  adminForm!: FormGroup;
 
   constructor(
     private route: Router,
-    private dialogService: DialogService,
+    private questionService: QuestionService,
+    private modalService: NgbModal,
+    public toastService: ToastService,
     private fb: FormBuilder
   ) {}
 
-  public formSubmitted() {
+  public formSubmitted(formValue: any) {
     let userdata = this.userData?.find(
-      (value: any) => value?.email ==  this.adminForm?.value?.email && value?.password ==  this.adminForm?.value?.password 
+      (value: any) =>
+        value?.email == this.adminForm?.value?.email &&
```

Comment:
if formValue is passed here then why this.adminForm?.value?.email need to use like this ?